java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3250,4 +3250,25 @@ public void bitcount(BaseClient client) {
                 executionException.getCause().getMessage(),
                 "An error was signalled by the server - ResponseError: syntax error");
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void getbit(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        String missingKey = "missing";
+        String value = "foobar";
+
+        assertEquals(OK, client.set(key, value).get());
+        assertEquals(1, client.getbit(key, 1).get());
+        assertEquals(0, client.getbit(key, 1000).get());
```

Comment:
Can you add a test where response is not 1 or 0?

Reply:
I think the response is only 1, 0, or and exception.